src/common/actions/create-snap.js

```diff
@@ -8,33 +8,75 @@ const BASE_URL = conf.get('BASE_URL');
 
 export const SET_GITHUB_REPOSITORY = 'SET_GITHUB_REPOSITORY';
 export const CREATE_SNAP = 'CREATE_SNAP';
+export const CREATE_SNAP_SUCCESS = 'CREATE_SNAP_SUCCESS';
 export const CREATE_SNAP_ERROR = 'CREATE_SNAP_ERROR';
 
+// XXX cjwatson 2017-02-08: Hardcoded for now, but should eventually be
+// configurable.
+export const STORE_SERIES = '16';
+export const STORE_CHANNELS = ['edge'];
+
 export function setGitHubRepository(value) {
   return {
     type: SET_GITHUB_REPOSITORY,
     payload: value
   };
 }
 
+// Just enough to satisfy higher-level code that might receive errors from
+// our internal API or errors thrown directly from here.
+function throwAPICompatibleError(payload) {
```

Comment:
Have you considered creating a sub-class of Error instead? It might be more idiomatic.

```
class APICompatibleError extends Error {
  constructor(payload) {
    this.message = ...
    this.json = ....
  }
}

throw new APICompatibleError(payload);
```